operators/hpe-csi-operator/destinations/certified-operators/current-version/manifests/hpe-csi-operator.clusterserviceversion.yaml

```diff
@@ -155,30 +201,30 @@ spec:
             path: disableNodeConfiguration
             x-descriptors:
               - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
-          - description: 'Disable NodeGetVolumeStats call to CSI driver. (default: false)'
+          - description: 'Disable NodeGetVolumeStats call to CSI driver (default: false)'
             displayName: Disable NodeGetVolumeStats RPC
             path: disableNodeGetVolumeStats
             x-descriptors:
               - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
-          - description: iSCSI Configuration
-            displayName: iSCSI Configuration
-            path: iscsi
-          - description: Username for iSCSI CHAP authentication
-            displayName: iSCSI CHAP username
-            path: iscsi.chapUsername
+          - description: 'Disable the Node Monitor (default: false)'
+            displayName: Disable Node Monitor
+            path: disableNodeMonitor
             x-descriptors:
-              - urn:alm:descriptor:com.tectonic.ui:text
-          - description: Password for iSCSI CHAP authentication
-            displayName: iSCSI CHAP password
-            path: iscsi.chapPassword
+              - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+          - description: 'Disables host deletion by the CSP when no volumes are associated with the host (default: false)'
```

Comment:
This will apply only for the CHAP enabled volumes. We will delete the host definition for other volumes even if the flag is 'false'. Can we change this to "Disable host deletion by the CSP when no CHAP enabled volumes are associated with the host" or a better way to clear this.

Reply:
Can we check with @sijeesh if that's really the case? I thought this would be a broad stroking option?

Reply:
That's true @datamattsson . This flag can be used for any host, regardless of whether the linked volumes use CHAP or not.